### PR TITLE
Teacher's Digital Platform: Update Financial literacy activities finder results count to 25 per page

### PR DESCRIPTION
**Pull request by anselmbradford**
See https://[GHE]/CFPB/el-camino/issues/365

## Changes

- Teacher's Digital Platform: Update Financial literacy activities finder results count to 25

## How to test this PR

1. Search on http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/ should have 25 results or less per page. Compare to 5 results or less on production https://www.consumerfinance.gov/consumer-tools/educator-tools/youth-financial-education/teach/activities/

### CODE REVIEW

**Comment by anselmbradford** [comment]
Holding till we figure out if we want a smaller results count for pages with tall results items.

File: cfgov/teachers_digital_platform/models/activity_index_page.py [hunk 1]
```diff
@@ -393,13 +393,13 @@ def validate_results_per_page(request):
     A utility for parsing the requested number of results per page.
 
     This should catch an invalid number of results and always return
-    a valid number of results, defaulting to 5.
+    a valid number of results, defaulting to 25.
     """
     raw_results = request.GET.get('results')
     if raw_results in ['10', '25', '50']:
```
Comment (willbarton) [inline cfgov/teachers_digital_platform/models/activity_index_page.py:399]: If it defaults to 25, we could probably remove 25 here. But I don't think it's required to merge this PR.

**Review by willbarton — APPROVED** [review]

**Comment by sonnakim** [comment]
@anselmbradford I checked with the stakeholder, and she wants to leave the number of results at 5 per page. I've left a comment on our internal repo.

**Comment by anselmbradford** [comment]
Closing per request to keep the count at 5

**Pull request closed without merging**